src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -1058,12 +1061,14 @@ export default class QuestionnaireForm extends Component {
               Next
             </button>
           )} */}
-          <button
+           {/*
```

Comment:
We can probably add this button back if we're not doing a demo anytime soon

Reply:
Do we want the button to be in the final checked in version?